test/parallel/test-http2-ping-settings-heapdump.js

```diff
@@ -11,6 +11,14 @@ const v8 = require('v8');
 // after it is destroyed, either because they are detached from it or have been
 // destroyed themselves.
 
+// We use an higher autoSelectFamilyAttemptTimeout in this test as the v8.getHeapSnapshot().resume()
+// will slow the connection flow and we don't want the second connection attempt to start.
+let autoSelectFamilyAttemptTimeout = common.platformTimeout(1000);
+if (common.isWindows) {
+  // Some of the windows machines in the CI need more time to establish connection
+  autoSelectFamilyAttemptTimeout = common.platformTimeout(10000);
```

Comment:
It seems we should have a way for setting `autoSelectFamilyAttemptTimeout` globally, or possibly increase this timeout for everybody.

Reply:
Not a problem, I'll add it to the PR. But this time only and API, no CLI switch. Do you agree?

Reply:
@mcollina I've added the new global getter and setter. For now no CLI option but I doubt it might be really useful. Worst case I can add it later.